src/common/scrub_types.cc

```diff
@@ -161,6 +161,19 @@ void inconsistent_obj_wrapper::encode(bufferlist& bl) const
   ENCODE_FINISH(bl);
 }
 
+bufferlist inconsistent_obj_wrapper::encode() const
```

Comment:
Have this one call the out-param version.  I *really* don't want two copies of encode.